src/test/java/api/loans/DueDateScheduledNoticesProcessingTests.java

```diff
@@ -384,11 +403,28 @@ void testNoticeIsNotSentOrDeletedWhenPatronNoticeRequestFails() {
 
     checkSentNotices();
 
-    verifyNumberOfScheduledNotices(1);
+    verifyNumberOfScheduledNotices(0);
     verifyNumberOfPublishedEvents(NOTICE, 0);
     verifyNumberOfPublishedEvents(NOTICE_ERROR, 1);
   }
 
+  @Test
+  void testErrorEventsAreSentWhenUpdateNoticeRequestFails() {
```

Comment:
Test that
- notice is not sent
- event is not published
- notice is deleted

Reply:
Test deleted, as we cannot access fake storage instances